packages/world/src/systems/libraries/MoveLib.sol

```diff
@@ -158,7 +168,7 @@ library MoveLib {
       return false;
     }
 
-    (, ObjectTypeId belowObjectTypeId) = belowCoord.getOrCreateEntity();
+    (, ObjectTypeId belowObjectTypeId) = belowCoord.getEntity();
```

Comment:
in spawns, moves I realized we dont need to create the air entity, its unnecessary gas. But forgot to update gravity which is also a move, updated here